.travis.yml

```diff
@@ -60,4 +60,4 @@ script:
 - if [[ "$PHPUNIT" == "1" ]]; then composer test; fi
 - if [[ $TRAVIS_PHP_VERSION == "5.6" || $TRAVIS_PHP_VERSION == "7.4" ]]; then composer validate --no-check-all; fi
 # Check for known security vulnerabilities in the currently locked-in dependencies.
-- if [[ "$SECURITY" == "1" ]]; then php $SECURITYCHECK_DIR/security-checker.phar -n security:check $(pwd)/composer.lock;fi
+$SECURITYCHECK_DIR/local-php-security-checker_1.0.0_linux_amd64 --path=$(pwd)/composer.lock;fi
```

Comment:
The if condition got inadvertently removed here. 

The committed `composer.lock` file is the one which needs to be checked, but as that file shouldn't really change during CI builds (except when testing PHP 8, but the security check not be enabled for that run), the check only needs to be executed once.

As things stand, this should be failing with:
1. an parse error in the Travis script (the `;fi` at the end of the line not closing an open `if`)
2. an error for the script not existing when the condition is not met as it wouldn't have been downloaded.

Reply:
even though I love your suggestion, I'd rather not go through 11 repositories again unless it's broken.
also I've fixed the script :-) well spotted!